nucleus/admin/server-mgmt/pom.xml

```diff
@@ -162,5 +162,17 @@
             <groupId>org.glassfish.annotations</groupId>
             <artifactId>logging-annotation-processor</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>1.62</version>
```

Comment:
I'm not sure about this particular artifact but `org.bouncycastle` stuff usually occurs in multiple modules so it maybe should be manged using dependency management?

Reply:
It is used in javaee7-samples, but not in payara, but it could be (as a test dependency).
But generally I agree, it is good idea to put it to dependency-management at least to have versions in one place.